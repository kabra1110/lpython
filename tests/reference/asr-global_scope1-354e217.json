{
    "basename": "asr-global_scope1-354e217",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/global_scope1.py",
    "infile_hash": "e7b33a2680429ee47d15af5af7c31ac24cc802d18d5566b568fc8ef4",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-global_scope1-354e217.stdout",
    "stdout_hash": "09607beca31352c346dad19af56cc5705e1c59b3686b575cf254a29e",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}