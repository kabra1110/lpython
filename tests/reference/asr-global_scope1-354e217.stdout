(TranslationUnit
    (SymbolTable
        1
        {
            __main__:
                (Module
                    (SymbolTable
                        2
                        {
                            __main____global_statements:
                                (Function
                                    (SymbolTable
                                        3
                                        {
                                            
                                        })
                                    __main____global_statements
                                    (FunctionType
                                        []
                                        ()
                                        Source
                                        Implementation
                                        ()
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        []
                                        .false.
                                    )
                                    []
                                    []
                                    [(=
                                        (Var 2 i)
                                        (IntegerConstant 5 (Integer 4))
                                        ()
                                    )]
                                    ()
                                    Public
                                    .false.
                                    .false.
                                    ()
                                ),
                            i:
                                (Variable
                                    2
                                    i
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    __main__
                    []
                    .false.
                    .false.
                ),
            main_program:
                (Program
                    (SymbolTable
                        4
                        {
                            __main____global_statements:
                                (ExternalSymbol
                                    4
                                    __main____global_statements
                                    2 __main____global_statements
                                    __main__
                                    []
                                    __main____global_statements
                                    Public
                                )
                        })
                    main_program
                    [__main__]
                    [(SubroutineCall
                        4 __main____global_statements
                        2 __main____global_statements
                        []
                        ()
                    )]
                )
        })
    []
)
